src/urllib3/connection.py

```diff
@@ -327,12 +327,6 @@ def request_chunked(
         self.send(b"0\r\n\r\n")
 
 
-_PCTRTT = Tuple[Tuple[str, str], ...]
```

Comment:
Moved those to `util/ssl_.py` because both `connection.py` and `util/ssltransport.py` need them.